src/_includes/graphql/customer-orders-output.md

```diff
@@ -0,0 +1,343 @@
+`match` | String | Defines a filter that performs a fuzzy search on the specified string. For example, if you specify a value of `20`, the query returns all order IDs that contain the string `20`
+
+### orders output attributes (CustomerOrders) {#customerOrders}
+
+The `CustomerOrders` object contains the results of the filter defined in the `orders` attribute.
+
+Attribute | Data type | Description
+--- | --- | ---
+`items` | [[CustomerOrder]!](#customerOrder) | An array of items in an order
+`page_info` | [SearchResultPageInfo](#SearchResultPageInfo) | An object that includes the `current_page`, `page_info`, and `page_size` values specified in the query
+`total_count` | Int | The total count of customer orders
+
+### CustomerOrder attributes {#customerOrder}
+
+The `CustomerOrder` object contains details about each order returned by the `orders` attribute.
+
+Attribute | Data type | Description
+--- | --- | ---
+`billing_address` | [CustomerAddress](#customerAddressOutput) | The billing address for the order
+`carrier` | String | The shipping carrier for the order delivery
+`comments` | [[CommentItem](#CommentItem)] | Comments on the order
+`created_at` | String | Deprecated. Use the `order_date` attribute instead
+`credit_memos` | [[CreditMemo](#CreditMemo)] | Contains a list of credit memos for the order
+`grand_total` | Float  | Deprecated. Use the `totals.grand_total` attribute instead
+`id` | ID! | Unique identifier for the order
+`increment_id` | String | Deprecated. Use the `id` attribute instead
+`invoices` | [[Invoice](#Invoice)]! | Contains a list of invoices for the order
+`items` | [[OrderItemInterface](#OrderItemInterface)] | An array containing the items purchased in this order
+`number` | String! | The order number
+`order_date` | String! | The date the order was placed
+`order_number` | String! | Deprecated. Use the number `attribute` instead
+`payment_methods` | [[PaymentMethod](#PaymentMethod)] | Payment details for the order
+`shipments` | [[OrderShipment](#OrderShipment)] | Shipment list for the order
+`shipping_address` | [CustomerAddress](#customerAddressOutput) | shipping address for the order
+`shipping_method` | String | Shipping method for the order
+`status` | String! | The current status of the order
+`total` | [OrderTotal](#OrderTotal) | Contains details about the calculated totals for this order
+
+The deprecated attributes were previously defined in the `CustomerOrder` object in the `customerOrders` query, but have been deprecated for the `customer` query:
+
+#### BundleInvoiceItem {#BundleInvoiceItem}
+
+The `BundleInvoiceItem` object implements the `InvoiceItemInterface`. It also defines the following attribute:
+
+Attribute | Data type | Description
+--- | --- | ---
+`bundle_options` | [[ItemSelectedBundleOption]](#ItemSelectedBundleOption) | A list of bundle options that are assigned to the bundle product
+
+#### BundleOrderItem {#BundleOrderItem}
```

Comment:
Do you want BundleOrderItem to have the same format as BundleInvoiceItem, with the description and table?